lib/kubeclient/common.rb

```diff
@@ -253,16 +254,18 @@ def watch_entities(resource_name, options = {})
       uri = @api_endpoint.merge("#{@api_endpoint.path}/#{@api_version}/#{path}")
 
       params = {}
+      format = options.delete(:raw) ? :text : :json
```

Comment:
please elaborate a bit on this part. deleting a passed option without visibility to whoever called the code?